docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+# THIS IS TO BE USED FOR DEVELOPMENT AND TESTING PURPOSES ONLY
```

Comment:
Could you document the role of this file in the test and/or link to a README documenting the testing process?

Reply:
Also, could this file move to a test directory?